### ISSUE TITLE
[SUGG] Change Health HUD bar

### ISSUE THREAD

**Issue by Delnth** [HaloSpaceStation/HaloSpaceStation13#1862]
Title. Replace the current bay health HUD bar that displays one's health state for one like TG or any other similar system.
Currently, someone may even have 300 brute damages, they will still appear with yellow vitals, which are... "minor" injuries, orange being medium and red being major, blinking red/black is critical. It makes it hard for a medic role such as Huragoks, where they must scan everyone, which is a waste of time if there is a dying victim that needs immediate treatment.

I like the way the vitals look like, but I don't like how they work. They aren't accurate at all. It'd be nice if they could be made as accurate as possible. And preferably clearly visible, because the current vitals are kinda hard to see sometimes.

**Comment by X0-11** [comment]
it's based off of pain

**Comment by Delnth** [comment]
It should be changed then. It is just way too inaccurate that it makes such mechanic unreliable. When you have 300 brutes but have yellow vitals, then you are either on very strong painkillers or something's really wrong.

If it was based off missing health, it'd be way better.

**Comment by X0-11** [comment]
it's based off of pain because it's meant to be what your character feels. The downside of not being able to easily ascertain your health state at a glance is the intended downside of painkillers

**Comment by Delnth** [comment]
I'm talking from the perspective of someone using the health HUD. And honestly, it is both useless and unreliable. Being unable to ascertain one's health state makes incredibly difficult for whoever is using the HUD to tend to the most critically injured players.

Let's said a sangheili rushed three marines. They are all down. One has 300 burn damages, one has 150 and the other has 80-90 burn damages. There will be absolutely no difference between those three. They will either have orange or yellow vitals.

When you are on-site, you see three severely injured marines. You know your HUD scans for pain, but do you need the HUD to know that? Absolutely not, because they are unconscious after all the plasma that burned their body parts. Naturally, they will be in pain. This is why it makes the HUD useless.

Being able to know who's the most injured at a glance would vastly increase the healing effectiveness of medic roles because they wouldn't have to check the casualties one by one no more, thus wasting precious seconds that they could've have spent tending to a critically injured player.

**Comment by X0-11** [comment]
oh wait I missed the point: it's the medglasses you're on about. Improving the accuracy of those would probably be a good idea.

**Comment by Delnth** [comment]
Yeah. All I'm suggestive is a bit (a lot) more visibility for the "health bar" ontop of mobs and better health tracking. Because as state previously, you simply cannot differentiate someone that has 300 brute and someone that has 90 brute. They will likely if not certainly both have yellow vitals.

The critical state (blinking red and black) is never seen because of brain med (probably why there is poor health tracking too, since it checks a lot of brain damage rather than body)

**Comment by reklaw9** [comment]
Health HUDs see pulse (blue for sedated/normal and stabilized,green is for normal pulse, yellow is for irregular pulse, orange is a lot of pain/very irregular pulse, red and blinking is no pulse,black is for ded/non-organic heart), what i recommend is copying (or doing something like) the TG HealthHud but with color to show pulse and maybe even a yellow health icon(the normal one is red) for people that are with less than 85% of blood oxygenation.
